gprofiler/main.py

```diff
@@ -183,7 +183,7 @@ def _generate_output_files(
     ) -> None:
         start_ts = get_iso8601_format_time(local_start_time)
         end_ts = get_iso8601_format_time(local_end_time)
-        base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts))
+        base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts.replace(":", "-")))
```

Comment:
Let's do the replace only if Windows, easy trick is:
```suggestion
        base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts.replace(":", "-" if is_windows() else ":")))
```